main.js

```diff
@@ -0,0 +1,11 @@
+var saveButton = document.querySelector("button");
+var titleText = document.querySelector("#title");
+var bodyText = document.querySelector("#body");
+var outputContainer = document.querySelector(".output-container");
+
+
+var ideas = {};
+
+saveButton.addEventListener("click", function() {
+    console.log("this button is clicked");
```

Comment:
@ben-rosner-williamsburg I'm assuming this `console.log` is temporary and will be removed in a later PR?

Reply:
@ericahagle Yep. I've removed it already.